wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack.swift

```diff
@@ -235,7 +235,7 @@ public class CoreDataStack: NSObject, ContextProvider {
                     return
                 }
                 if tp.warnIfLongerThanInterval() == false {
-                    WireLogger.localStorage.debug("time spent in migration only: \(tp.elapsedTime)")
+                    WireLogger.localStorage.debug("time spent in migration only: \(tp.elapsedTime)", attributes: .safePublic)
```

Comment:
Is the `debug` level filtered out for production builds (although the safe public attribute) is set?

Reply:
no the safePublic here is for os_log to not redact the log

Reply:
actually you're right I'll increase to info